EIPS/eip-draft_nft_updatable.md

```diff
@@ -0,0 +1,233 @@
+                            "required": ["eval"],
+                            "properties": {
+                                "eval": {
+                                    "type": "string"
+                                    "description": "The evaluation formula to transform the last JSON metadata using the engine above (can take arguments)",
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+"NFT Metadata Updates JSON Schema" : 
+
+```json
+{
+    "title": "Metadata Updates JSON Schema",
+    "type": "object",
+    "properties": {
+        "updates": {
+            "type": "array",
+            "description": "A list of updates to apply sequentially to calculate updated metadata"
+            "items": { "$ref": "#/$defs/update" }
+            "$defs": {
+                "update": {
+                    "type": "object",
+                    "required": ["tokenId", "recipeKey"],
+                    "properties": {
+                        "tokenId": {
+                            "type": "string",
+                            "description": "The tokenId for which the update recipe should apply"
+                         },
+                        "recipeKey": {
+                            "type": "string",
+                            "description": "recipeKey to use to get the JSON transformation expression in current metadata"
+                        },
+                        "args": {
+                            "type": "string",
+                            "description": "arguments to pass to the JSON transformation"
+                        }
+                    }
+                 }
+            }
+        }
+    }
+}
+```
+
+## Engines
+
+Only one engine is currently defined in this extension proposal.
+
+If the engine in the original metadata is "jsonata@1.8.*", updated metadata is calculated starting from original metadata and applying each update sequentially (all updates which are present in all the URIs emitted by the event `MetadataUpdates` for which tokenId matches).
+
+For each step, the next metadata is obtained by the javascript calculation (or compatible jsonata implementation in other language) :
+
+```js
+const nextMetadata = jsonata(evalString).evaluate(previousMetadata, args)
+```
+
+With `evalString` is found with `recipeKey` in the original metadata recipes list.
+
+If the key is not present in the original metadata list, `previousMetadata` is kept as the valid updated metadata.
+
+If the evaluation throws any errors, `previousMetadata` is kept as the valid updated metadata.
+
+If a validation Schema JSON has been defined and the result JSON `nextMetadata` does not conform, that update is not valid and `previousMetadata` is kept as the valid updated metadata.
+
```

Comment:
This is not a valid EIP section. Please move this content as a subsection of a section defined in https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md